tools/launch_ccm_cluster.py

```diff
@@ -287,7 +287,7 @@ def run_script(scriptname, args = []):
                 stdout = sys.stdout
                 sys.stdout = sys.stderr
                 script_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), scriptname)
-                subprocess.check_call(['bash', script_path] + args)
+                subprocess.check_call(['bash', script_path] + args, stdout=sys.stdout)
```

Comment:
Could the surrounding redirect be removed in favor of setting `stdout=sys.stderr` here?